public/app/features/dimensions/utils.ts

```diff
@@ -42,6 +44,21 @@ export function getScaleDimensionFromData(
   return getScaledDimension(undefined, cfg);
 }
 
+export function getScalarDimensionFromData(
```

Comment:
This missing function tripped me up a little bit, kept trying to use getScaleDimension which was resulting in `NaN` being returned